test/JDBC/pg_hint_plan/BABEL-3291.sql

```diff
@@ -0,0 +1,26 @@
+-- Drop the table if it exists already and create a new table
+drop table if exists babel_3291_t1
+go
+
+create table babel_3291_t1(a1 int PRIMARY KEY, b1 int)
+go
+
+-- Switch the session variable to show the query plan to ON
+set babelfish_showplan_all on
+go
+
+-- Run a SELECT query without any hints. The query plan will use an index scan in this case
```

Comment:
Not to nitpick, but the comment doesn't mention what we'd discussed: that it's critical that this query plan is not what's hinted at. I suggest something like:

```
/*
 * Run a query with no hints. Besides verifying that un-hinted queries still
 * work, this test has a VERY important side-effect: it ensures that when this
 * query is not hinted it produces a *different* plan than the sequential scan
 * that we're hinting below. This is crucial because it's the only way to
 * verify that the next test is actually valid. IE: if the planner was going to
 * choose a sequential scan anyway, the next test wouldn't actually prove that
 * hints were working.
 */
```